packages/web-shared/components/VideoPlayer/VideoPlayer.js

```diff
@@ -35,11 +35,13 @@ function VideoPlayer(props = {}) {
   const [paused, setPaused] = useState(false);
 
   // will find the first HLS video playlist provided
-  const videoMedia = props.parentNode?.videos?.find(video =>
-    video.sources?.some(source => source.uri.includes("youtube.com"))
-  ) || props.data?.videos?.[0];
-
-
+  const hlsMedia = props.parentNode?.videos?.find(
+    (video) => video.sources.length && video.sources[0].uri.includes('.m3u8')
```

Comment:
Should this be video.sources?.length ?

Reply:
And uri?.includes ?

Reply:
We could? But this is how it was before the bad PR. Sources should always be at least an empty array here

Reply:
Video.sources.length is new. 

prior it was video.sources?.some or other checks that wouldn’t throw if video.sources was undefined. 

also looks like prior code had || props.data?.videos?.[0]; but that is completely gone now? Looking at my phone now.